src/cdk/v2/destinations/dynamic_yield/procWorkflow.yaml

```diff
@@ -0,0 +1,108 @@
+bindings:
+  - name: EventType
+    path: ../../../../constants
+  - path: ../../../../v0/destinations/dynamic_yield/config
+  - name: removeUndefinedAndNullValues
+    path: ../../../../v0/util
+  - path: ../../bindings/jsontemplate
+
+steps:
+  - name: messageType
+    template: |
+      .message.type.toLowerCase()
+  - name: validateInput
+    template: |
+      let messageType = $.outputs.messageType;
+      $.assert(.message.type, "message Type is not present. Aborting message.");
```

Comment:
should we do validation before messageType step

Reply:
Done